articles/machine-learning-and-deep-learning/index.md

```diff
@@ -0,0 +1,55 @@
+
+Machine learning uses Artificial Intelligence (AI) techniques that assist in decision making. Conventional algorithms in AI aid in performing mathematical computation and predictions. Machine learning depends on fed structured data. Conventional algorithms aids in achieving data mining and predictive analytics in a search.
+
+Unlike machine learning, AI can use [unstructured data](https://www.datamation.com/big-data/structured-vs-unstructured-data.html) and still be able to function. Deep learning does mimic the human brain's workings. This is because it uses this data to perform the processing of data. Machine learning aids in the detection of objects, language translation, and decision making. Machine learning is also able to recognize speech and make prompt decisions. The AI makes this possible this allows in no aspect of human supervision.
+
+### Deep Learning
+
+Deep learning is a subset of machine learning. Deep learning uses networks that operate unsupervised and one that uses [structured data](https://www.datamation.com/big-data/structured-vs-unstructured-data.html). This is like the human brain structure as it operates as a subset of machine learning.
+
+This is to means that the algorithms used are complex and are from neural networks. This resemblance is like the layers with neural connections in the human brain.
+
+### How deep learning works
+
+Deep learning uses algorithms whose functionality is like that of machine learning. One major distinction on each layer is that it has a different interpretation of data. The [neural networks](https://news.mit.edu/2017/explained-neural-networks-deep-learning-0414) learn where data is fetched.
+
+The neurons in the network undertake two tasks. The first includes understanding which feature is optimal for performing a specific task. The second one involves classifying data after many iterations of the learning process. In case there is a mistake, the deep learning model's accuracy increases. Deep learning is, thus, a more complex and more capable version of machine learning.
+
+### Machine Learning vs. Deep Learning
+
+Deep learning differs from machine learning in some aspects. One major attribute is that deep learning learns from its mistakes. In most cases, this is achieved without human intervention. As for machine learning, there is a need for a human to correct the mistake made.
+
+Deep learning then adopts an algorithm then learns not to make that mistake again in the future. This enhances the [multi-level neural networks](https://towardsdatascience.com/multi-layer-neural-networks-with-sigmoid-function-deep-learning-for-rookies-2-bf464f09eb7f?gi=e7b1f9430ac5) to learn from their mistakes. But, this model does not guarantee success or no mistakes in the future. This because a wrong prediction can be made depending on the quality of the data fed to them. The accuracy is dependent on the quality of data used to learn.
+
+### Applications and Conclusion
```

Comment:
mentioning the applications in the introduction and discussing them in-depth would be a great alternative to enhance readability.